packages/console/src/pages/ApiResourceDetails/index.tsx

```diff
@@ -156,41 +123,28 @@ const ApiResourceDetails = () => {
             )}
           </Card>
           <TabNav>
-            <TabNavItem href={location.pathname}>{t('general.settings_nav')}</TabNavItem>
+            <TabNavItem href={`/api-resources/${data.id}/${ApiResourceDetailsTabs.Settings}`}>
+              {t('api_resource_details.settings_tab')}
+            </TabNavItem>
+            <TabNavItem href={`/api-resources/${data.id}/${ApiResourceDetailsTabs.Permissions}`}>
+              {t('api_resource_details.permission_tab')}
+            </TabNavItem>
           </TabNav>
-          <DetailsForm
-            isDirty={isDirty}
-            isSubmitting={isSubmitting}
-            onDiscard={reset}
-            onSubmit={onSubmit}
-          >
-            <FormCard
-              title="api_resource_details.settings"
-              description="api_resource_details.settings_description"
-              learnMoreLink="https://docs.logto.io/docs/recipes/protect-your-api"
-            >
-              <FormField isRequired title="api_resources.api_name">
-                <TextInput
-                  {...register('name', { required: true })}
-                  hasError={Boolean(errors.name)}
-                  readOnly={isLogtoManagementApiResource}
-                  placeholder={t('api_resources.api_name_placeholder')}
-                />
-              </FormField>
-              <FormField isRequired title="api_resource_details.token_expiration_time_in_seconds">
-                <TextInput
-                  {...register('accessTokenTtl', { required: true, valueAsNumber: true })}
-                  hasError={Boolean(errors.accessTokenTtl)}
-                  placeholder={t(
-                    'api_resource_details.token_expiration_time_in_seconds_placeholder'
-                  )}
-                />
-              </FormField>
-            </FormCard>
-          </DetailsForm>
+          <Outlet
+            context={
+              // eslint-disable-next-line no-restricted-syntax
+              {
+                resource: data,
+                isDeleting,
+                isLogtoManagementApiResource,
+                onResourceUpdated: (resource: Resource) => {
+                  void mutate(resource);
+                },
+              } as ApiResourceDetailsOutletContext
+            }
+          />
```

Comment:
Why Outlet? I don't quite get it

Reply:
Per offline discussion, use `Outlet` to handle tabs.